.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
   - 2.2.3
+before_script:
+  - sed -f _scripts/baseurl.sed _config.yml > _scripts/_config.yml
+  - mv _scripts/_config.yml _config.yml 
```

Comment:
Would a comment in _config.yml suffice instead of writing a new file? It seems like a comment would be less crufty and vastly more explicit than the implicit message sent by redirecting output to file with "deploy" in the name.